tests/utils/Statistic/min-intervals.test.js

```diff
@@ -0,0 +1,23 @@
+import { describe, expect, test } from 'vitest'
+import getMinInterval from '$lib/utils/Statistic/min-interval';
+import data from './min-intervals-data.json' assert { type: 'json' };
+
+describe("Find smallest interval", () => {
```

Comment:
structurally it feels like it should be:
Describe `Find smallest interval`
- test `should return smallest interval without provided StatType (default to history)`
- test `should return smallest interval between forecasts`
- test `should return smallest interval between histories`

Reply:
I meant putting the three `tests` inside one `describe`.